harmonytypes/v1/types.proto

```diff
@@ -115,6 +115,45 @@ message Attachment {
   string caption = 5;
 }
 
+/*
+  Minithumbnail is an extremely low-quality JPEG thumbnail.
+
+  The resolution is usually no larger than 64x64.
```

Comment:
64x64 is too small, and images can vary resolution ratios. We should put a file size limit instead like 100KB.

Reply:
100KB is far too big; considering that the JPEG data is inlined in the message itself.

The purpose of a minithumbnail is supposed to be something that the client can use to get a blob of colours to display blurrily, like so:

Reply:
![Screenshot_20210813-210123](https://user-images.githubusercontent.com/20326855/129429927-c7b3b19c-ca55-4e34-8fc2-b0bbcd167433.png)

Reply:
Well it should at least match the width and height of the image, so we can remove the width and height from the Minithumbnail and move the data field into Photo itself.

Reply:
no, the minithumbnail needs to be extremely small, otherwise you bloat the message with too much photo data

Reply:
Well, if we want to display it in message history, once the actual photo loads, the messages will jump up and down. So the width and height should match the actual images as in it's ratio, ie. if it's 16:9 it should stay 16:9.

Reply:
64x64 is the recommended maximum; doesn't mean messages have to be 1:1 ratio